indico/modules/events/papers/controllers/display.py

```diff
@@ -59,6 +59,12 @@ class RHPaperTimeline(RHPaperBase):
     def _process(self):
         return render_paper_page(self.paper, view_class=WPDisplayCallForPapers)
 
+    def _check_paper_protection(self):
+        return (self.contribution.is_user_associated(session.user, check_abstract=True) or
+                self.paper.cfp.is_manager(session.user) or
```

Comment:
I'd use `self.event_new.cfp` simply to make it clear that this is not related to that specific paper